Docs: Add guidelines for running Grafana behind proxy on RPM-based Linux (#41710)
grafana/grafana: docs/sources/installation/rpm.md

{{< docs/shared "systemd/bind-net-capabilities.md" >}}

#### Serving Grafana behind a proxy

> When serving grafana behind a proxy you need to set `http_proxy` and `https_proxy` environment variables.

Reviewer: ```suggestion
When serving Grafana behind a proxy, you need to configure the `http_proxy` and `https_proxy` environment variables.
```

##### Centos 6

> If you run Centos 6, to do so add the following lines in `/etc/sysconfig/grafana-server`.

Reviewer: ```suggestion
If you are on Centos 6, add the following lines to the `/etc/sysconfig/grafana-server` file.
```

```
export no_proxy=internal.domain,127.0.0.1
export http_proxy=http://proxy.domain:3128/
export https_proxy=http://proxy.domain:3128/
```

##### Centos 7

> If you run Centos 6, to do so add the following lines in `/etc/sysconfig/grafana-server`.

Reviewer: ```suggestion
If you are on Centos 7, add the following lines to the `/etc/sysconfig/grafana-server` file.
```
